samples/04.2-modify-transferprocess/modify-transferprocess-sample-integration-tests/src/test/java/org/eclipse/dataspaceconnector/samples/sample042/test/ModifyTransferProcessSampleTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial test implementation for sample
+ *
+ */
+
+package org.eclipse.dataspaceconnector.samples.sample042.test;
+
+import io.restassured.RestAssured;
+import org.apache.http.HttpStatus;
+import org.eclipse.dataspaceconnector.common.util.junit.annotations.EndToEndTest;
+import org.eclipse.dataspaceconnector.junit.extensions.EdcRuntimeExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.time.Duration;
+import java.util.Map;
+
+import static org.awaitility.Awaitility.await;
+import static org.eclipse.dataspaceconnector.extension.sample.test.FileTransferSampleTestCommon.API_KEY_HEADER_KEY;
+import static org.eclipse.dataspaceconnector.extension.sample.test.FileTransferSampleTestCommon.API_KEY_HEADER_VALUE;
+import static org.eclipse.dataspaceconnector.extension.sample.test.FileTransferSampleTestCommon.TRANSFER_PROCESS_URI;
+import static org.eclipse.dataspaceconnector.extension.sample.test.FileTransferSampleTestCommon.getFileFromRelativePath;
+import static org.hamcrest.Matchers.equalTo;
+
+@EndToEndTest
+public class ModifyTransferProcessSampleTest {
+
+    static final String CONSUMER_CONFIG_PROPERTIES_FILE_PATH = "samples/04.2-modify-transferprocess/consumer/config.properties";
+    public static final String EXPECTED_ID_PROPERTY_NAME = "id[0]";
```

Comment:
why `public` on these? are they ref'd anywhere else?

Reply:
Most of the constants are anyways only used once. I suggest inlining those.

Reply:
Good catch @chrislomonico! This happens by using the `Introduce constant` refactoring tool from IntelliJ and the author not double-checking the result!

I will make it consistent to package-protected as the others.